src/hooks/useCustomWallpaper.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+
+const WALLPAPER_PATH = '/Photos/Settings/Wallpaper.jpg'
+
+const useCustomWallpaper = client => {
+  const [wallpaperLink, setWallpaperLink] = useState(null)
+  const [fetchStatus, setFetchStatus] = useState('idle')
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setFetchStatus('loading')
+        const response = await client
+          .collection('io.cozy.files')
+          .getDownloadLinkByPath(WALLPAPER_PATH)
+        setWallpaperLink(response)
+        setFetchStatus('loaded')
+      } catch (error) {
```

Comment:
lint error ?

Reply:
error is not used?

Reply:
No, but that's not a linting error, at least in the default configuration https://eslint.org/docs/rules/no-unused-vars#caughterrors-none